Sources_v3/APIClient/Endpoints/Payloads/UserPayloads.swift

```diff
@@ -7,7 +7,7 @@ import Foundation
 enum UserPayloadsCodingKeys: String, CodingKey {
     case id
     case role
-    case name
+    case name // TODO: remove name and avatarURL below, since they're parsed as extraData
```

Comment:
Wouldn't it be easier to just delete them instead of this todo?

Reply:
Oh yeah, that TODO was actually for myself to not forget 😅

Reply:
Glad it worked out

Reply:
Fixed in 66b7f4c514244952de084ec846f05d090f03f866